src/tests/bundles/noble-caracal.yaml

```diff
@@ -26,6 +26,11 @@ applications:
     channel: latest/edge
     constraints: "virt-type=virtual-machine mem=8G"
 
+  grafana-agent:
+    charm: ch:grafana-agent
+    channel: latest/edge
```

Comment:
Is there a more stable channel available?

Reply:
we could use `latest/stable`. Is there a reason to break from the convention of other charms in the test bundle that are all `latest/edge`?

Reply:
I see what you mean and thanks for updating the channel, I'll share my perspective:
* The cos-lite bundle added in the same commit use all stable charms.
* From our charm gate point of view the cos charms are more of a test fixture, and I think we would benefit by de-risking breaks due to random changes to other projects charms (OTOH, we likely would want to know about said breakages from some source).